app/src/main/java/com/framgia/soundclound/screen/local/TrackLocalViewModel.java

```diff
@@ -12,6 +12,7 @@
 import com.framgia.soundclound.BR;
 import com.framgia.soundclound.data.model.Track;
 import com.framgia.soundclound.data.source.TrackRepository;
+import com.framgia.soundclound.data.source.local.SharePreferences;
```

Comment:
ERROR: Unused import: com.framgia.soundclound.data.source.local.SharePreferences. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports